loguru.hpp

```diff
@@ -2301,6 +2319,30 @@ namespace loguru
 
 	// ------------------------------------------------------------------------
 
+	static void print_preamble_header(char* out_buff, size_t out_buff_size)
+	{
+		long pos = 0;
+		snprintf(out_buff, out_buff_size, ""); // Make sure there is a '\0' and handle out_buff_size==0
+ 		if(g_preamble_date) {
```

Comment:
Please follow the Loguru code style with a space after if (`if (g_preamble_date) {`) here and elsewhere